src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -2086,6 +2086,91 @@ .locals init (int V_0)
 ");
         }
 
+        [Fact]
+        public void EventFields_Attributes()
```

Comment:
@tmat How complete do you think the set of tests for these things should be?

My feeling is that if we verify that attributes on a specific table as emitted then its safe to skip all of the scenarios (adding, deleting, adding something deleted in a delta, deleting something added in a delta etc.) but given that there is a known gap (in parameter attributes) I'm not too sure. Could at least add something for event accessors I guess?

Reply:
What was the gap for parameters?

I'd think that compiler tests that cover all tables that may have attributes should be sufficient.

You can also combe adding + updating + deleting attribute in a single test.

Reply:
IDE tests should be focused on attributes on symbols (including attribute target specifications like `return:`). Compiler tests should be focused on CA target metadata tables.

Reply:
> What was the gap for parameters?

For parameters (and I'm expecting it for type parameters) the issue was that they were never emitted beyond generation 0, except for newly created methods, which is essentially what my feeling is based on. ie, once I made it so that updated parameters were emitted, the custom attribute handling worked without issue.